cmake/Reprojucer.cmake

```diff
@@ -3609,6 +3609,9 @@ function(_FRUT_set_compiler_and_linker_settings target)
       target_compile_options(${target} PRIVATE
         "-mmacosx-version-min=${osx_deployment_target}"
       )
+      target_link_options(${target} PRIVATE
+        "-mmacosx-version-min=${osx_deployment_target}"
+      )
```

Comment:
`target_link_options` was recently introduced in CMake 3.13, but the minimum CMake version required for FRUT is 3.4. Please replace with
```cmake
      set_property(TARGET ${target} APPEND PROPERTY LINK_FLAGS
        "-mmacosx-version-min=${osx_deployment_target}"
      )
```